src/components/Picker/BasePicker/index.js

```diff
@@ -1,30 +1,72 @@
 import React from 'react';
 import RNPickerSelect from 'react-native-picker-select';
+import {View} from 'react-native';
+import _ from 'underscore';
 
 import styles from '../../../styles/styles';
 import * as basePickerPropTypes from './basePickerPropTypes';
 import basePickerStyles from './basePickerStyles';
 
-const BasePicker = props => (
-    <RNPickerSelect
-        onValueChange={props.onChange}
-        items={props.items}
-        style={props.size === 'normal' ? basePickerStyles(props.disabled, props.hasError, props.focused) : styles.pickerSmall}
-        useNativeAndroidPickerStyle={false}
-        placeholder={props.placeholder}
-        value={props.value}
-        Icon={() => props.icon(props.size)}
-        disabled={props.disabled}
-        fixAndroidTouchableBug
-        onOpen={props.onOpen}
-        onClose={props.onClose}
-        pickerProps={{
-            onFocus: props.onOpen,
-            onBlur: props.onClose,
-        }}
-    />
-);
+class BasePicker extends React.Component {
+    constructor(props) {
+        super(props);
 
+        this.state = {
+            selectValue: this.props.value || this.props.defaultValue,
+        };
+    }
+
+    componentDidMount() {
+        const viewRef = this.viewRef;
+        if (!viewRef) {
+            return;
+        }
+
+        if (typeof this.props.innerRef !== 'function') {
+            return;
+        }
+
+        // 'ref' is not exposed in Picker to focus on it. An alternative is getting a View container ref. View ref will return a <div>,
+        // apply 'focus()' on it will not scroll to it. Applying 'scrollIntoView(false)' will scroll to element. https://developer.mozilla.org/es/docs/Web/API/Element/scrollIntoView
+        const originalFocus = viewRef.focus;
+        viewRef.focus = function () {
+            // Execute original focus() using apply on ref context to prevent infinite focus() call stack and preserve original focus() functionality
+            originalFocus.apply(viewRef);
+            viewRef.scrollIntoView(false);
+        };
+        this.props.innerRef(viewRef);
+    }
```

Comment:
As alternative to `Picker` ref for the porpose to scroll, I wrap `RNPickerSelect` component on a `View` component. I get this `View` ref instead of `RNPickerSelect` and extend its `focus` method with `scrollIntoView`.

Reply:
Thanks for the alternative solution @LucioChavezFuentes, but [your PR for RNPicker](https://github.com/react-native-picker/picker/pull/376) is almost merged. So hopefully we shouldn't be needing it

Reply:
Yes, I hope so too, I write it just in case there is urgency on fixing the issue.